android_core/src/test/kotlin/com/walletconnect/android_core/crypto/managers/KeyChainMock.kt

```
@@ -8,11 +8,11 @@ import com.walletconnect.util.hexToBytes
 internal class KeyChainMock : KeyStore {
     private val mapOfKeys = mutableMapOf<String, String>()
 
-    override fun setSymmetricKey(tag: String, key: Key) {
+    override fun setKey(tag: String, key: Key) {
         mapOfKeys[tag] = key.keyAsHex
     }
 
-    override fun getSymmetricKey(tag: String): String {
+    override fun getKey(tag: String): String {
         return mapOfKeys[tag] ?: ""
```

looking at this, we should not be defaulting to an empty string. If a tag does not exist, we should throw an exception

---

I think this should be part of https://github.com/WalletConnect/WalletConnectKotlinV2/issues/368